com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/HDShaderGUI.cs

```diff
@@ -34,6 +45,11 @@ public override void AssignNewShaderToMaterial(Material material, Shader oldShad
             SetupMaterialKeywordsAndPassInternal(material);
         }
 
+        /// <summary>
+        /// Setup the keywords and passes for the material. It is required to call this function after changing a property on a material to ensure it's validity.
+        /// </summary>
+        /// <param name="changed">GUI.changed is the usual value for this parameter. If changed is false, the function will just exit.</param>
```

Comment:
GUI.changed is the usual value for this parameter. If this value is false, the function just exits.